lib/spear/reading.ex

```diff
@@ -79,6 +79,16 @@ defmodule Spear.Reading do
     |> map_all_position()
   end
 
+  # coveralls-ignore-start
+  defp map_all_position(%Spear.Event{
+         metadata: %{link: %{commit_position: commit, prepare_position: prepare}}
```

Comment:
coveralls ignores on these because projected streams aren't enabled yet in CI

should really turn them on so we can test behavior lik this :upside_down_face: